composer/cicd_sample/README.md

```diff
@@ -1,5 +1,5 @@
 # Composer CI/CD Sample
 
-Code in this directory is part of a tutorial found [in the Cloud docs](https://cloud.google.com/composer/docs/dag-cicd-integration-guide). In this tutorial, you will learn how to use Cloud Build to sync DAGs from version control with your Cloud Composer environment.
+Code in this directory is part of a tutorial found [in the Cloud docs](https://cloud.google.com/composer/docs/composer-2/dag-cicd-github). In this tutorial, you will learn how to use Cloud Build to sync DAGs from version control with your Cloud Composer environment.
 
 While the DAGs found in this tutorial specifically are written for use in Airflow 2, this tutorial can be used in Airflow 1, Airflow 2, Composer 1, and Composer 2 environments.
```

Comment:
Does `Airflow 2` also mean `Composer 2`?

Reply:
Confusingly enough, they are not synonymous - this line is fine as is 😁